meshroom/ui/reconstruction.py

```diff
@@ -235,15 +237,20 @@ def _updateSfMParams(self):
             self._reconstructed = self._R is not None
         self.sfmParamsChanged.emit()
 
-    def _updateDenseSceneParams(self):
-        """ Update internal members depending on PrepareDenseScene. """
+    def _updateUndistortedImageParams(self):
+        """ Update internal members depending on PrepareDenseScene or ExportAnimatedCamera. """
         # undistorted image path
-        if not self._activeNode_PrepareDenseScene.node:
+        if not self._activeNode_PrepareDenseScene.node or not self._activeNode_ExportAnimatedCamera.node:
+        # if not self._activeNode_PrepareDenseScene.node:
             self._undistortedImagePath = ''
+        if self._activeNode_ExportAnimatedCamera.node:
+            basename = "{}_{}".format(self._viewpoint.intrinsicId.value, os.path.basename(self._viewpoint.path.value))
+            filename = "{}.{}".format(os.path.splitext(basename)[0], self._activeNode_ExportAnimatedCamera.node.undistortedImageType.value)
+            self._undistortedImagePath = os.path.join(self._activeNode_ExportAnimatedCamera.node.output.value, "undistort", filename)
```

Comment:
It would be better to update the "outputUndistorted" param with the right expression and the semantic flag:
https://github.com/alicevision/Meshroom/blob/develop/meshroom/nodes/aliceVision/ExportAnimatedCamera.py#L109
And then use it here.